content-src/components/DiscoveryStreamComponents/DSCard/DSCard.jsx

```diff
@@ -53,7 +59,11 @@ export const VariantMeta = ({
     </div>
     {cta && <button className="button cta-button">{cta}</button>}
```

Comment:
If by mistake server json  sets CTA for non-paid content this will display both button and messaging which doesn't look right.

Lets make CTA button variant fail proof by showing button only for spoc content something like

{context && cta && <button className="button cta-button">{cta}</button>}